internal/common/schema/pagination_schema.go

```diff
@@ -0,0 +1,34 @@
+package schema
```

Comment:
recommend to change package name to avoid conflicts with TF packages, e.g. dsschema, schemads, or similar